spec/support/authentication_helpers.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module AuthenticationHelpers
-  mattr_accessor :username, :password, :token
+  mattr_accessor :username, :password
```

Comment:
The token is only used for SDR API operations, so move that code into the deposit helpers.